scalanet/ut/src/io/iohk/scalanet/peergroup/DynamicTLSPeerGroupSpec.scala

```diff
@@ -314,6 +312,51 @@ class DynamicTLSPeerGroupSpec extends AsyncFlatSpec with BeforeAndAfterAll {
         }
     }
   }
+  it should "handle messages with prepended messages id-s" in taskTestCase {
+    implicit val s = Codec[TestMessage[String]]
+    (for {
+      client <- DynamicTLSPeerGroup[TestMessage[String]](getCorrectConfig())
+      server <- DynamicTLSPeerGroup[TestMessage[String]](getCorrectConfig())
+      ch1 <- client.client(server.processAddress)
+    } yield (client, server, ch1)).use {
+      case (client, server, ch1) =>
+        for {
+          result <- Task.parZip2(
+            ch1.sendMessage(TestMessage.Foo("hello")),
+            server.serverEventObservable.collectChannelCreated.mergeMap {
+              case (channel, release) =>
+                channel.channelEventObservable.guarantee(release)
+            }.headL
+          )
+          (_, eventReceived) = result
+        } yield {
+          assert(eventReceived == MessageReceived(TestMessage.Foo("hello")))
+        }
+    }
+  }
+
+  it should "inform user about too large frame" in taskTestCase {
+    (for {
+      client <- DynamicTLSPeerGroup[Byte](getCorrectConfig())
+      // 1byte message + 4 bytes length field gives frame of size 5, so maxFrameLength = 4 should end with error
```

Comment:
I assume these are the settings in `getCorrectConfig()`, right? 
Could you add them to the constructor here, so that if the defaults change this test doesn't get affected?

Reply:
Sorry, I meant adding them to the `client <- DynamicTLSPeerGroup[Byte](getCorrectConfig(maxFrameLength=5, lenghtFieldLength=4))` so that the what the comment says are not just assumption knowing what the test setup is, but always forces it to be what this test needs.